app/scripts/lib/app-start.js

```diff
@@ -60,6 +60,7 @@ define([
   'models/auth_brokers/iframe',
   'models/unique-user-id',
   'models/user',
+  'models/verification/same-browser',
```

Comment:
nit: we just "git" rid of `same browser` in https://github.com/mozilla/fxa-content-server/commit/094bfdf2f619f14b9666f200d68f80ac573380b1

Is this something we want to rename here?